lib/wordpress.js

```diff
@@ -0,0 +1,48 @@
+var _ = require('underscore');
+var params = require('querystring').parse(window.location.search.split('?').pop());
+
+module.exports = {
+  getPostPreviewView: function (view) {
+    // Return false if we don't have everything we need for a preview
+    var isPreviewActive = params._wp_json_nonce && params.is_preview;
+    if (!isPreviewActive) {
+      console.info('Preview is not active.');
+      return false;
+    }
+
+    // Create preview object
+    if (_.isFunction(view)) {
+      view = {template: view};
+    }
+    if (_.isFunction(view.preprocessor)) {
```

Comment:
If there's no `view` ∴ no template, then we don't have all of the arguments we need to return a usable `view` to pass into `render()`. I can't imagine a scenario where we'd call `getPostPreviewView()` without any arguments.